### ISSUE TITLE
[BUG]  accessibility issue

### ISSUE THREAD

**Issue by priyankarpal** [rupali-codes/LinksHub#442]
### Is there any existing issue for this?

- [X] I have searched the existing issues

### Description

There are some issues regarding accessibility :

- Buttons do not have an accessible name
- Background and foreground colors do not have a sufficient contrast ratio.
- Lists do not contain only `<li>` elements and script-supporting elements (`<script>` and `<template>`).

### Screenshots

- ![image](https://user-images.githubusercontent.com/88102392/232233138-04d6928c-d628-4792-99a4-f8a143c743df.png)

- ![image](https://user-images.githubusercontent.com/88102392/232233152-ea5b7154-af4b-4053-b824-27e8e0969947.png)

- ![image](https://user-images.githubusercontent.com/88102392/232233158-518b457a-d6aa-4bff-8313-dfc6356ee28e.png)

**Comment by rupali-codes** [comment]
Hey Priyankar, could please explain in details?

**Comment by priyankarpal** [comment]
You can check these problems using the lighthouse tool. These are minor problems if we fix them it will be good for SEO.

**Comment by rupali-codes** [comment]
Sure, thanks

**Comment by EmmaDawsonDev** [comment]
I can add more details:
- The dark/light mode button lacks an accessible name. It currently gets read out by a screen reader as just "button". This can be fixed by adding `aria-label="toggle dark mode`. It should also have an `aria-pressed=true/false` attribute which toggles when the button is pressed so that the user knows if dark mode is activated or not.
- The hamburger menu also lacks an accessible name and is announced only as "button". This can be fixed by adding `aria-label=navigation`. It should also have an attribute of `aria-expanded=true/false` which toggles when the navigation is open or closed so that a screen reader user knows if the navigation is open or not.
- The navigation menu is an unordered list but that can only have list items as children. The first child of the navigation list is a div (unsure of its purpose) which is causing the error.

**Comment by rupali-codes** [comment]
Thanks a lot @EmmaDawsonDev for pointing out this, will surely work on this :)
